web/pdf_page_view.js

```diff
@@ -142,6 +142,25 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @private
+   */
+  async _renderAnnotationLayer() {
+    let error = null;
+    try {
+      await this.annotationLayer.render(this.viewport, "display");
+      error = false;
+    } catch (e) {
+      error = e;
+    } finally {
+      this.eventBus.dispatch("annotationsrender", {
```

Comment:
Please call this `annotationlayerrendered` instead, since that's consistent with the event used in the text-layer.

Reply:
Yes, `annotationlayerrendered` is an even better name than my `annotationsrendered` suggestion above. Let's use this instead.